### DISCUSSION TITLE
Open in a new tab - 'Link to a document' SharePoint content type

### DISCUSSION THREAD

**Discussion by JoeAyre** [microsoft-search/pnp-modern-search#2654] [Q&A] [answered]
Hi,

Hoping someone can help. I'm using the 'Details list' layout within the PnP - Search Results web part and am surfacing SharePoint content of content type 'Link to a document'. 

I have a slot name 'FilePath' that uses the slot field 'Path'.
I also have a slot name 'Title' that uses the slot field 'Title'.

In the 'Manage columns' option within the web part I have the following 'Column value' for 'Title'

```
<a href="{{slot item @root.slots.FilePath}}?web=0" target="_blank" style="color: {{@root.theme.semanticColors.link}}">
	{{slot item @root.slots.Title}}
</a>
```

Unfortunately, when a user clicks on one of the links, the URL opens up in the same tab, but the user cannot use the back button to get back to the previous page.

Does anyone know of a way to either:

a) force the link to open in a new tab, or
b) if a) is not possible, ensure the user can use the back button to return to the previous page?

Thanks,
Joe

**Comment by wobba** [comment, marked as answer]
This is a SPFx issue. Add `data-interception="off"` as well to the tag and it should work. See https://learn.microsoft.com/en-us/sharepoint/dev/spfx/hyperlinking

**Reply by JoeAyre** [reply]
Perfect @wobba - really appreciate your help on this. I can confirm that this solved my issue. Thanks, Joe
